internal/app/wwctl/configure/ssh/root.go

```diff
@@ -12,12 +12,14 @@ var (
 			"keys.",
 		RunE: CobraRunE,
 	}
+	keyTypes []string
 )
 
 func init() {
 }
 
 // GetRootCommand returns the root cobra.Command for the application.
 func GetCommand() *cobra.Command {
+	baseCmd.PersistentFlags().StringArrayVarP(&keyTypes, "keytypes", "t", []string{"rsa", "dsa", "ecdsa", "ed25519"}, "ssh key types to be created")
```

Comment:
Can this value also be specified in `warewulf.conf`? If not, I don't think this could be done with `wwctl configure -a`. And wouldn't it need to be specified every time you run `wwctl configure`?

Reply:
Also, please add documentation for it. :)

Reply:
Its now in the delivered `warewulf.conf` which is also the documentation for all this stuff.